telcell/data/parsers.py

```diff
@@ -42,7 +48,10 @@ def parse_measurements_csv(path: Union[str, Path]) -> List[Track]:
                 Measurement(
                     lat=float(row['celldb.wgs84.lat']),
                     lon=float(row['celldb.wgs84.lon']),
-                    timestamp=row['timestamp'],
+                    timestamp=datetime.datetime.strptime(
```

Comment:
Deze kan ik niet plaatsen; ik verwacht
`timestamp = datetime.datetime.fromisoformat(row['timestamp'])`

Sowieso is timezone info nodig om de timestamp te kunnen duiden.

Reply:
Ik had deze parser gebaseerd op de `.csv` die nu in onze test folder leeft. Deze heeft blijkbaar een ander formaat dan de standaard `measurements.csv`... Ik zal de test data en code aanpassen naar het juiste formaat.